cli/cmd/install.go

```diff
@@ -49,6 +49,7 @@ type (
 		controllerUID               int64
 		disableH2Upgrade            bool
 		disableHeartbeat            bool
+		heartbeatScheduleStringVal  string
```

Comment:
Shouldn't this just be `heartbeatSchedule`?

Reply:
heartbeatSchedule is already a func() (L66) hence the awkward name. I'd be happy to reverse the naming logic (`heartbeatSchedule` for the variable, `getHeartbeatSchedule()` for the function) if people like.